draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1015,6 +1039,47 @@ Instances of these functions are given in {{suites}}, which defines a list of
 suites that specify a full set of parameters matching elliptic curves and
 algorithms.
 
+## Domain separation requirements {#domain-separation}
+
+When invoking hash\_to\_curve, implementors MUST use domain separation
+({{term-domain-separation}}) to avoid interfering with other protocols
+that also use the hash\_to\_curve functionality.
+In addition, any protocol that uses two or more hash\_to\_curve functions
+targeting different elliptic curves MUST enforce domain separation between
```

Comment:
Is the expectation that if one had two hash-to-curve functions H1 and H2 targeting the *same* curve, then H1 = H2?

Reply:
Yes, unless they're explicitly separated. I suppose I should clarify this point.

Reply:
I pushed an edit to address this.